arch/arm/src/stm32h7/stm32h7x3xx_rcc.c

```diff
@@ -886,7 +886,19 @@ void stm32_stdclockconfig(void)
           STM32_PWR_CR3_SMPSEXTHP | STM32_PWR_CR3_SMPSLEVEL_MASK);
       regval |= STM32_PWR_CR3_SCUEN;
       putreg32(regval, STM32_PWR_CR3);
+#elif defined(CONFIG_STM32H7_PWR_LDO_SUPPLY)
+      regval = getreg32(STM32_PWR_CR3);
+      regval &= ~(STM32_PWR_CR3_BYPASS | STM32_PWR_CR3_SCUEN |
+          STM32_PWR_CR3_SMPSEXTHP | STM32_PWR_CR3_SMPSLEVEL_MASK);
+      regval |= STM32_PWR_CR3_LDOEN;
+      putreg32(regval, STM32_PWR_CR3);
 #else
+      /* 4 else cases of power supply mode
```

Comment:
There are 3 settings. 2 are covered above.

What is the "4 else cases of power supply mode" referring too. This it the third. It is the default STM32H7_PWR_DEFAULT_SUPPLY.  should there be another else with #error "No a STM32 H7 Power Supply Selection made"?

Reply:
@davids5
STM32H7 has six power supply modes to choose from.
We use LDO supply.
I suspect someone is using a DIRECT SMPS supply.
And I don't know what the first person used.
So we don't know what the 'else' power mode is.

'else' is considered to be when nothing is defined, but I don't know how to change 'default' from 'defining nothing' in 'Kconfig'.

What do you think we should do?